src/components/Details/Legislation/LegislativeHistoryNode.tsx

```diff
@@ -13,54 +14,57 @@ import { Dot, DOT_SIZE } from "../../Shared/Dot";
 import useFetchData, {
   initialFetchDataState,
 } from "../../../containers/FetchDataContainer/useFetchData";
-import { FetchDataContainer } from "../../../containers/FetchDataContainer";
+import LazyFetchDataContainer from "../../../containers/FetchDataContainer/LazyFetchDataContainer";
 
 import styled from "@emotion/styled";
 import Details from "../../Shared/Details";
 import { strings } from "../../../assets/LocalizedStrings";
+import { screenWidths } from "../../../styles/mediaBreakpoints";
 export interface LegislativeHistoryNodeProps {
   /** event in the matter's timeline */
   eventMinutesItem: EventMinutesItem;
   /** this node is the last index on the timeline */
   isLastIndex: boolean;
 }
 
-const EVENT_MINUTES_ITEM_DECISION_COLOR = {
-  [EVENT_MINUTES_ITEM_DECISION.PASSED]: "cdp-bg-acceptance-green",
-  [EVENT_MINUTES_ITEM_DECISION.FAILED]: "cdp-bg-rejected-red",
-};
-const MARGIN = 12;
-
-const ReferenceFrame = styled.div({ position: "relative" });
-const ExpandingContainer = styled.div({
-  margin: MARGIN,
-  display: "flex",
-  flexDirection: "row",
-  gap: 32,
-});
-const TextContainer = styled.div({ flex: 1 });
-const VotingGraphicContainer = styled.div({ flex: 1, marginTop: MARGIN, marginLeft: MARGIN * 2 });
-const TitleBox = styled.div({ display: "flex", flexDirection: "row", cursor: "pointer" });
-
-const ConnectBar = styled.div({
-  position: "absolute",
-  top: MARGIN + DOT_SIZE,
-  left: DOT_SIZE / 2 + MARGIN,
-  height: "100%",
-  width: 2,
-  zIndex: 0,
-  backgroundColor: "grey",
-});
-
 const LegislativeHistoryNode: FC<LegislativeHistoryNodeProps> = ({
   eventMinutesItem,
   isLastIndex,
 }: LegislativeHistoryNodeProps) => {
+  const isMobile = useMediaQuery({ query: `(max-width: ${screenWidths.tablet})` });
+
+  const EVENT_MINUTES_ITEM_DECISION_COLOR = {
+    [EVENT_MINUTES_ITEM_DECISION.PASSED]: "cdp-bg-acceptance-green",
+    [EVENT_MINUTES_ITEM_DECISION.FAILED]: "cdp-bg-rejected-red",
+  };
+  const MARGIN = isMobile ? 4 : 12;
+
+  const ReferenceFrame = styled.div({ position: "relative" });
+  const ExpandingContainer = styled.div({
+    margin: MARGIN,
+    display: "flex",
+    flexDirection: "row",
+    gap: MARGIN * 3,
+  });
+  const TextContainer = styled.div({ flex: 1 });
+  const VotingGraphicContainer = styled.div<{ marginLeft: number }>((props) => ({
+    flex: 1,
+    marginTop: MARGIN,
+    marginLeft: props.marginLeft,
```

Comment:
Just a suggestion: you could add the media query here
```
const VotingGraphicContainer = styled.div({
    flex: 1,
    marginTop: MARGIN,
   marginLeft: 0, //on mobile
   [`@media (min-width:${screenWidths.tablet})`]: {
         marginLeft: MARGIN * 2 //on tablet or above
  },

```
I think this is more succient to keep the styling logic in one place.